specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,76 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for humans and systems. The FOCUS specification does not require a specific level of precision for numeric values. The level of precision required for a given column is determined by the provider and should be part of a data definition published by the provider.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST follow the formatting requirements listed below. Provider-generated numeric value capturing columns SHOULD adopt backward-compatible format requirements over time.
```

Comment:
In other attributes, we're going with language like this: 
"All columns capturing a date/time value, defined in the FOCUS specification, MUST follow the formatting requirements listed below. Provider-generated, date/time-related columns SHOULD adopt the same format requirements over time."

Should we make the others look like this OR make this consistent with the others? Fine either way. I don't feel like we need to explicitly say backward-compatible FWIW. How providers solve these issues for their customers is up to them. They might create new data sets for all we know.

Reply:
I think I proposed the backward-compatibility when reviewing the PR, because the original statement looked too strong to me. Sorry, I had not cross checked the language used in other language attributes. I do not have a strong opinion about stating explicitly backward compatibility and I am fine to make this consistent with others.